### PR TITLE
docs: Improved language in main README.md

### PR DESCRIPTION
**Pull request by quinton-hoole-2**
Made some improvements to the grammar and flow of the intro doc.

**What type of PR is this?**
 /kind documentation

**What this PR does / why we need it**:
Improves documentation.

**Which issue(s) this PR fixes**:
None

**Special notes for your reviewer**:
None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,35 +6,35 @@
...
+KubeEdge is an open source system extending native containerized application orchestration and device management to hosts at the Edge. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. It also supports **MQTT** and allows developers to author custom logic and enable resource constrained device communication at the Edge. Kubeedge consists of a cloud part and an edge part.  The edge part has been already open sourced and the cloud part is coming soon!
 
 ## Advantages
 
 #### Edge Computing
 
-With business logic running at Edge, volumes of data can be secured & processed locally. It reduces the bandwidth request between Edge and Cloud; increases the response speak; and protects customers' data privacy. 
+With business logic running at the Edge, much larger volumes of data can be secured & processed locally where the data is produced. Tihs reduces the network bandwidth requirements and consumption between Edge and Cloud. This increases responsiveness, decreases costs, and protects customers' data privacy.
```
Comment (rohitsardesai83) [inline README.md]: nit: Tihs -> This

File: README.md [hunk 2]
```diff
@@ -6,35 +6,35 @@
...
-#### Simplify development
+#### Simplified development
 
-Developers can write regular http or mqtt based applications; containerize and run anywhere at Edge or Cloud.
+Developers can write regular http or mqtt based applications, containerize these, and run them anywhere  - either at the Edge or in the Cloud - whichever is more appropriate.
 
 #### Kubernetes-native support
 
-With KubeEdge, users can orchestrate apps, manage devices and monitor app/device status against Edge nodes like a normal K8s cluster in the Cloud
+With KubeEdge, users can orchestrate apps, manage devices and monitor app and device status on Edge nodes just like a traditional kubernetes cluster in the Cloud
```
Comment (rohitsardesai83) [inline README.md]: nit: kubernetes -> Kubernetes

**Comment by rohitsardesai83** [comment]
Some minor comments , but LGTM overall. Thanks @quinton-hoole-2  for your contribution !

**Comment by m1093782566** [comment]
@sids-b 

Could you have someone take a look at the failing test case?

```
Summarizing 1 Failure:
[Fail] Event Bus Testing Publish on eventbus topics throgh MQTT internal broker [It] TC_TEST_EBUS_6: change the device status to online from eventbus 
/home/travis/gopath/src/github.com/kubeedge/kubeedge/edge/test/integration/device/device_test.go:121
```

File: README.md [hunk 3]
```diff
@@ -6,35 +6,35 @@
 
 <img src="./docs/images/KubeEdge_logo.png">
 
-KubeEdge is an open source system extending native containerized application orchestration and device management to hosts at Edge. It is built upon Kubernetes and provides core infrastructure support for network, app. deployment and metadata synchronization between cloud and edge. It also supports **MQTT** and allows developers to author customer logic and enable resource constraint devices communication at Edge. Basically kubeedge consists of cloud part and edge part, the edge part has been already open sourced and cloud part is coming soon!
+KubeEdge is an open source system extending native containerized application orchestration and device management to hosts at the Edge. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. It also supports **MQTT** and allows developers to author custom logic and enable resource constrained device communication at the Edge. Kubeedge consists of a cloud part and an edge part.  The edge part has been already open sourced and the cloud part is coming soon!
```
Comment (sids-b) [inline README.md]: nit: double spaces (before start of last sentence)

Comment (sids-b) [reply README.md]: nit: The edge part has been already open sourced -> The edge part has already been open sourced

File: README.md [hunk 4]
```diff
@@ -6,35 +6,35 @@
...
 #### Edge Computing
 
-With business logic running at Edge, volumes of data can be secured & processed locally. It reduces the bandwidth request between Edge and Cloud; increases the response speak; and protects customers' data privacy. 
+With business logic running at the Edge, much larger volumes of data can be secured & processed locally where the data is produced. Tihs reduces the network bandwidth requirements and consumption between Edge and Cloud. This increases responsiveness, decreases costs, and protects customers' data privacy. 
 
-#### Simplify development
+#### Simplified development
 
-Developers can write regular http or mqtt based applications; containerize and run anywhere at Edge or Cloud.
+Developers can write regular http or mqtt based applications, containerize these, and run them anywhere  - either at the Edge or in the Cloud - whichever is more appropriate.
```
Comment (sids-b) [inline README.md]: nit: double spaces (present before - either)

File: README.md [hunk 5]
```diff
@@ -43,32 +43,31 @@ KubeEdge is composed of these components:
...
+- Support more types of device protocols to Edge nodes such as AMQP, BlueTooth, ZigBee, etc.
+- Evaluate and enable much larger scale Edge clusters with thousands of Edge nodes and millions of devices.
+- Enable intelligent scheduling of applications to large scale Edge clusters.
 
 ## Usage
 
 ### Prerequisites
 
-To use KubeEdge, make sure you have **docker** in your environment, if don't have, please reference the following steps to install docker.
+To use KubeEdge, you will need to have  **docker** installed. If you don't, please follow these steps to install docker.
```
Comment (sids-b) [inline README.md]: nit: double spaces ( present before **docker**)

**Comment by sids-b** [comment]
> @sids-b
> 
> Could you have someone take a look at the failing test case?

Looks like a flaky integration test, @pavan187 is investigating :)

File: README.md (hunk 1 above)
Comment (quinton-hoole-2) [reply README.md]: Done, thanks.

File: README.md (hunk 2 above)
Comment (quinton-hoole-2) [reply README.md]: Done, thanks.

File: README.md (hunk 3 above)
Comment (quinton-hoole-2) [reply README.md]: Done, thanks.

File: README.md (hunk 4 above)
Comment (quinton-hoole-2) [reply README.md]: Done, thanks.

File: README.md (hunk 5 above)
Comment (quinton-hoole-2) [reply README.md]: Done, thanks.

**Comment by quinton-hoole-2** [comment]
Thanks for the prompt and thorough review guys.  I've addressed all the comments and squashed to one commit. Feel free to merge when you're ready.

**Pull request merged**